### PR TITLE
Rework 'No Team' section to include matchmaking flow

### PR DESCRIPTION
**Pull request by cabeese**
Separate the two possible "journeys" that a user can take: join a team with people they know, or try to find others to play with. Change the team section in their profile page to make these paths clearer.

**Note:** blocked on #160

### CODE REVIEW

File: client/components/team/NoTeamMessage.jsx [hunk 1]
```diff
@@ -1,35 +1,72 @@
...
+              <Link to='/team/create'>
+                <Button fluid color='teal'
+                        content='Create a Team'
+                        icon='right arrow' labelPosition='right'/>
+              </Link>
+
+              Browse existing teams
+              <Link to='/team/join'>
+                <Button fluid color='teal'
+                        content='Join Specific a Team'
```
Comment (richardgolding) [inline client/components/team/NoTeamMessage.jsx]: Minor typo; "a specific team"

**Review by richardgolding — APPROVED** [review]
This looks great.

Is there a logic to the color choices?

File: client/components/team/NoTeamMessage.jsx (hunk 1 above)
Comment (cabeese) [reply client/components/team/NoTeamMessage.jsx]: Great catch. fixed

**Comment by cabeese** [comment]
@richardgolding thanks! No, zero logic to the color changes. I just tried to go with two colors that were different enough to me (the token colorblind person) without being too atrocious. I have no idea if I met that goal so I'm more than happy if you want to suggest (or make in another PR) any changes

**Pull request merged**